### ISSUE TITLE
StarGAN-JTN training CelebA and RaFD facial expression synthesis

### ISSUE THREAD

**Issue by tsly123** [yunjey/stargan#87]
Hi,
Thank you for sharing your work @yunjey 

I am training StarGAN-JNT CelebA and RaFD for facial expression synthesis on CelebA. The training hasn't done yet (110k/200k iters) but the facial expression synthesis task seems not to converge. From the image below, while the attribute outputs are good, at least you can see the attribute in the output, but the facial expression synthesis outputs (last 7 columns) are not much different. This could be due to the size of RaFD dataset (4,824 samples as in the paper, 7 labels without contempt).
```
python main.py --mode=train --dataset Both --image_size 256 --c_dim 5 --c2_dim 7 \
                 --sample_dir stargan_both/samples --log_dir stargan_both/logs \
                 --model_save_dir stargan_both/models --result_dir stargan_both/results
```
What is the solution to this, more training? How to do facial expression synthesis on celebA as in the paper? Could you share the StarGAN-JNT pre-trained model to my email **176064 at jnu dot ac dot kr**?

![7-images](https://user-images.githubusercontent.com/35265385/54874338-ca3c7c80-4e2c-11e9-8231-18d7771bd73a.jpg)

I run several versions of StarGAN-JNT and ogange one is 110k/200k iter which gives the above result.
![image](https://user-images.githubusercontent.com/35265385/54874397-ed1b6080-4e2d-11e9-94c6-aef957b42669.png)

Thank you very much.
tsly

**Comment by ujjawalcse** [comment]
Hi @tsly123 !
 Have you got the pretrained model you asked for?
Can you share me also if you got it?

**Comment by Mouna-Dhaouadi** [comment]
Hi @tsly123 

In case you have got the pretrained mode you; can you share it ?  : mounaadhaouadi@gmail.com
Thanks you

**Comment by cy19980615** [comment]
> Hi,
> Thank you for sharing your work @yunjey
> I am training StarGAN-JNT CelebA and RaFD for facial expression synthesis on CelebA. The training hasn't done yet (110k/200k iters) but the facial expression synthesis task seems not to converge. From the image below, while the attribute outputs are good, at least you can see the attribute in the output, but the facial expression synthesis outputs (last 7 columns) are not much different. This could be due to the size of RaFD dataset (4,824 samples as in the paper, 7 labels without contempt).
> python main.py --mode=train --dataset Both --image_size 256 --c_dim 5 --c2_dim 7 \
>                  --sample_dir stargan_both/samples --log_dir stargan_both/logs \
>                  --model_save_dir stargan_both/models --result_dir stargan_both/results
> 
> What is the solution to this, more training? How to do facial expression synthesis on celebA as in the paper? Could you share the StarGAN-JNT pre-trained model to my email 176064 at jnu dot ac dot kr?
> 
> I run several versions of StarGAN-JNT and ogange one is 110k/200k iter which gives the above result.
> 
> Thank you very much.
> tsly

hello could you share me with the RaFD dataset by mailbox：1647560307@qq.com i am Anxious to do graduation design for undergraduates，thanks very much！！！

**Comment by alarazin** [comment]
I have the same problem @tsly123. Could you solve it? 
Thanks

**Comment by ShisuiUzumaki** [comment]
@tsly123 If you got the pre-trained model, could you share it at bsef17m051@pucit.edu.pk

**Comment by SonNNguyen** [comment]
@tsly123 Im working on my school project could you share the pre-trained model for RaFD, my email is [sonnn.m21ict@st.usth.edu.vn](mailto:sonnn.m21ict@st.usth.edu.vn) . Thank you very much!
